zenodo/config.py

```diff
@@ -743,6 +752,8 @@ def _(x):
                 'zenodo.modules.records.serializers.bibtex_v1_response'),
             'application/x-datacite+xml': (
                 'zenodo.modules.records.serializers.datacite_v31_response'),
+            'application/x-datacite+xmlv41': (
```

Comment:
@lnielsen @krzysztof `application/x-datacite+xmlv41` or `application/x-datacite-v41+xml`

Reply:
I guess we go with `application/x-datacite-v41+xml`, which seems more proper